### ISSUE TITLE
sendto device selection is removed in gnome-bluetooth 42.rc

### ISSUE THREAD

**Issue by bobby285271** [elementary/gnome-bluetooth-contract#1]
### What Happened?

Sorry if this is not the correct place. But we actually [have `gnome-bluetooth-contract` packaged](https://search.nixos.org/packages?channel=unstable&show=pantheon.gnome-bluetooth-contract&from=0&size=50&sort=relevance&type=packages&query=gnome-bluetooth) in NixOS :crying_cat_face: 

sendto device selection is removed in [this commit](https://gitlab.gnome.org/GNOME/gnome-bluetooth/-/commit/239a53d90c8182a95af15b8601223e7feeab89f4) when gnome-bluetooth is ported to GTK4. I think `bluetooth-sendto <file-name>` should no longer work, and this contract will probably be affected.

There is a WIP pull request that tries to re-add the device selection [here](https://gitlab.gnome.org/GNOME/gnome-bluetooth/-/merge_requests/120), but I am not sure it can become part of the 42 final release. Maybe we need to look for some replacement or shall we maintain a legacy gnome-bluetooth?

### Steps to Reproduce

1. Build gnome-bluetooth 42.rc
2. Run `bluetooth-sendto <file-name>` - no window shows

### Expected Behavior

A window like this should appear in some older gnome-bluetooth release:

![bluetooth-sendto](https://user-images.githubusercontent.com/20080233/158093793-12770082-693e-4bfc-8e44-f6f375308682.png)

### OS Version

Other Linux

### Software Version

Compiled from git

**Comment by bobby285271** [comment]
Some updates:

With wingpanel-indicator-bluetooth 7.0.0, [specifically this PR](https://github.com/elementary/wingpanel-indicator-bluetooth/pull/130/files), the indicator will create a "Send Files via Bluetooth" contract in `~/.local/share/contractor/io.elementary.bluetooth.contract` as:

```
[Contractor Entry]
Name=Send Files via Bluetooth
Icon=bluetooth
Description=Send files to device…
Exec=io.elementary.bluetooth -f %F
MimeType=!inode;
```

From my testing that should cover the functionality of this package. I assume it should be fine for us to drop gnome-bluetooth-contract from Nixpkgs :eyes: 

/cc https://github.com/NixOS/nixpkgs/pull/240261

Edit: later I found this is likely already not in elementary OS 7 ISO?

**Comment by danirabbit** [comment]
Yeah I think this needs to be archived! Thanks for following up here
